client/my-sites/plan-features/scroller.jsx

```diff
@@ -0,0 +1,334 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+import { clamp, inRange, range, round } from 'lodash';
```

Comment:
Any reason not to use `Math.round`?

Reply:
Because of the precision option.
https://github.com/Automattic/wp-calypso/pull/31802/files#diff-0f553a746951360a62f1a6bf514b6c17R238